core/server/logging/PrettyStream.js

```diff
@@ -68,84 +68,77 @@ PrettyStream.prototype.write = function write(data) {
 
     logLevel = '\x1B[' + codes[0] + 'm' + logLevel + '\x1B[' + codes[1] + 'm';
 
-    // CASE: logging.request
-    if (data.req && data.res) {
-        _.each(data.req, function (value, key) {
-            if (['headers', 'query', 'body'].indexOf(key) !== -1 && !_.isEmpty(value)) {
-                bodyPretty += '\n' + colorize('yellow', key.toUpperCase()) + '\n';
-                bodyPretty += prettyjson.render(value, {}) + '\n';
-            }
-        });
+    // CASE: bunyan passes each plain string/integer as `msg` attribute (logging.info('Hey!'))
+    if (data.msg) {
+        bodyPretty += data.msg;
 
-        if (data.err) {
-            if (data.err.level) {
-                bodyPretty += colorize('yellow', 'ERROR (' + data.err.level + ')') + '\n';
-            } else {
-                bodyPretty += colorize('yellow', 'ERROR\n');
-            }
-
-            _.each(data.err, function (value, key) {
-                if (['message', 'context', 'help', 'stack'].indexOf(key) !== -1 && !_.isEmpty(value)) {
-                    bodyPretty += value + '\n';
-                }
-            });
-        }
-
-        output += format('[%s] %s %s %s (%s)\n',
+        output += format('[%s] %s %s\n',
             time,
             logLevel,
-            data.req.method,
-            data.req.originalUrl,
-            data.res.statusCode
+            bodyPretty
         );
-
-        if (this.mode !== 'short') {
-            output += format('%s\n', colorize('grey', bodyPretty));
-        }
     }
-    // CASE: logging.error (standalone error)
-    else if (data.err) {
-        _.each(data.err, function (value, key) {
-            if (_.isEmpty(value)) {
-                return;
-            }
+    // CASE: log objects in pretty JSON format
+    else {
+        // common format: 127.0.0.1 user-identifier user-id [10/Oct/2000:13:55:36 -0700] "GET /apache_pb.gif HTTP/1.0" 200 2326
+        // if all values are available we log in common format
+        // can be extended to define from outside, but not important
+        try {
+            output += format('%s %s [%s] "%s %s" %s %s\n',
+                logLevel,
```

Comment:
This looks really awesome 👍 
I'm not entirely sure that having 127.0.0.1 output when developing locally is useful? Maybe something to consider for later, maybe it's only useful to include in the JSON output for production?

Also note there is a :bug: where I am seeing `undefined` for `data.req.ip` when first starting the server and hitting the blog.